bgpd/bgp_vty.c

```diff
@@ -1860,6 +1878,32 @@ DEFUN (no_bgp_always_compare_med,
 	return CMD_SUCCESS;
 }
 
+
+DEFUN(bgp_default_ebgp_policy_rfc8212, bgp_default_ebgp_policy_rfc8212_cmd,
+      "bgp default-ebgp-policy-rfc8212",
```

Comment:
From a UI perspective, the RFC number isn't something everyone will have in the back of their head, so the option name (and help text) should be something more descriptive.

`bgp ebgp-requires-route-maps` maybe? With `Require in and out route-maps for eBGP peers (RFC8212)` as help text?